app/views/conversations/_conversation.html.erb

```diff
@@ -81,15 +81,29 @@
             <table class="w-full text-left">
               <tr>
                 <td class="pb-2 whitespace-nowrap">Conversation</td>
-                <td class="pb-2">$</td>
+                <td class="pb-2">$<%= conversation.total_cost / 100.0%></td>
```

Comment:
`number_to_currency(amount / 100.0)` and it'll make sure 0.5 gets the extra zero 0.50 and it'll do the $ and thousands separators. Likewise on the other ones within this file.

However, we'll lose fractions of a cent. What precision should we show? I think I'd propose for the conversation we use the default, which is precision: 2 if you don't specify. Maybe put a ~ in front of that so your screenshot would have looked like. And then for the input and output costs we use a precision of 5, which is what your screenshot happened to show. I think any more than that and it gets hard to great.

```
Conversation ~   $0.03
Input
Tokens           2102
Cost             $0.001051
Output
Tokens           1174
Cost             $0.001761
```
(I put the tilde at the end of "Conversation" rather than within the dollar `td` so that the number alignment still works.)